Makefile

```diff
@@ -48,12 +48,10 @@ docker-shell:
 .PHONY: lint-fix
 lint-fix:
 	ruff check --fix ./app
-	black ./app
```

Comment:
Same here: replacing `black` means adding the ruff formatter: https://github.com/ParkenDD/park-api-v3/blob/main/Makefile#L186